CRM/PCP/BAO/PCP.php

```diff
@@ -453,7 +453,7 @@ public static function buildPcp($pcpId, &$page, &$elements = NULL) {
       $page->_defaults['pcp_is_anonymous'] = 0;
 
       $page->add('text', 'pcp_roll_nickname', ts('Name'), array('maxlength' => 30));
```

Comment:
as @KarinG indicated, this also need to be changed to ```$page->addField('pcp_roll_nickname', ['entity' => 'ContributionSoft', 'context' => 'create']);```

Reply:
Yep - I don't want to scope-creep this to other fields